app/views/partials/card.scala.html

```diff
@@ -0,0 +1,15 @@
+@(fileNames: List[String], metadataType: String, isEditing: Boolean = true)
```

Comment:
I'd suggest passing the text into the partial template, instead of a parameter. That way the controllers can sort out what test should be displayed. Also the partial template holds the minimal amount of logic

Reply:
As suggested I am now passing the text into the partial template.